mautic/mautic
# Upgrade Doctrine

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This step upgrade the required packages of doctrine, a follow up PR will come to fix `composer.json` versions and upgrade to the next version

**CS Fixer is failing because I upgraded the package, once we get this PR reviewed I will run it, I did not run it so the files to be reviewed is minimal**



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Use the system in different parts including emails, reports, and contacts
3. Make sure it works
